package-lock.json

```diff
@@ -952,6 +952,11 @@
         "@bbc/psammead-styles": "^0.3.2"
       },
       "dependencies": {
+        "@bbc/gel-foundations": {
```

Comment:
Probably best to reset this file to latest, to avoid complexity in testing multiple things

Reply:
That is supposed to be updated at 
https://github.com/bbc/psammead/blob/a6c6cce9c47f949568eab70b53919848441807bf/packages/components/psammead-timestamp/package.json#L19-L21
We should not directly update the lock file

Reply:
Are you on the right npm? we use 6.9.0 on node 10.16.0 now

Reply:
<img width="478" alt="Screenshot 2019-06-20 at 10 38 34" src="https://user-images.githubusercontent.com/48688870/59830109-9eeffd80-9347-11e9-8c3a-8315a87fa725.png">

Reply:
This PR doesn't add a new dependancy, we shouldnt update the package-